src/game/SpellEffects.cpp

```diff
@@ -1151,7 +1151,7 @@ void Spell::EffectDummy(SpellEffectIndex eff_idx)
                 }
                 case 32027:                                 // Expedition Flare
                 {
-                    if (!m_CastItem)                        // 32029 = Expedition Preserver | 32030 = Expedition Scout
+                        // 32029 = Expedition Preserver | 32030 = Expedition Scout
                         m_caster->CastSpell(m_caster, (urand(0, 1) ? 32029 : 32030), true, m_CastItem);
```

Comment:
m_caster->CastSpell(m_caster, (urand(0, 1) ? 32029 : 32030), true)

- Drop CastItem altogether from this one, since CastItem is nullptr in this case and nullptr is the default it can be considered redundancy.